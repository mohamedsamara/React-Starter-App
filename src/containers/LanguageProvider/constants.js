/*
 *
 * LanguageProvider constants
 *
 */

export const SET_LOCALE = 'src/LanguageProvider/SET_LOCALE';
