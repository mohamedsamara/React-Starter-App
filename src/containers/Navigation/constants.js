/*
 *
 * Navigation constants
 *
 */

export const TOGGLE_DRAWER = 'src/Navigation/TOGGLE_DRAWER';
