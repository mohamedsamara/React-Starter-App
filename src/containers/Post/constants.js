/*
 *
 * Post constants
 *
 */

export const FETCH_POSTS_REQUEST = 'src/Post/FETCH_POSTS_REQUEST';
export const FETCH_POSTS_SUCCESS = 'src/Post/FETCH_POSTS_SUCCESS';
export const HANLDE_POSTS_FAILURE = 'src/Post/HANLDE_POSTS_FAILURE';
export const DELETE_POST_SUCCESS = 'src/Post/DELETE_POST_SUCCESS';
