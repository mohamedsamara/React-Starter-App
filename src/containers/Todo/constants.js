/*
 *
 * Todo constants
 *
 */

export const FETCH_TODOS_REQUEST = 'src/Dashboard/FETCH_TODOS_REQUEST';
export const FETCH_TODOS_SUCCESS = 'src/Dashboard/FETCH_TODOS_SUCCESS';
export const FETCH_TODOS_FAILURE = 'src/Dashboard/FETCH_TODOS_FAILURE';
