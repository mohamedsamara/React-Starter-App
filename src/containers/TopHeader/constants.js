/*
 *
 * TopHeader constants
 *
 */

export const OPEN_MENU = 'src/TopHeader/OPEN_MENU';
export const CLOSE_MENU = 'src/TopHeader/CLOSE_MENU';
