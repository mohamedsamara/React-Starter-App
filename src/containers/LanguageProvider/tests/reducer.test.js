import languageProviderReducer from '../reducer';
import { SET_LOCALE } from '../constants';

describe('languageProviderReducer', () => {
  let initialState;
  beforeEach(() => {
    initialState = {
      locale: 'en',
    };
  });

  it('should return the initial state', () => {
    const expected = initialState;
    expect(languageProviderReducer(undefined, {})).toEqual(expected);
  });

  it('should handle the SET_LOCALE action correctly', () => {
    const action = { type: SET_LOCALE };
    expect(languageProviderReducer(undefined, action)).toMatchSnapshot();
  });
});
