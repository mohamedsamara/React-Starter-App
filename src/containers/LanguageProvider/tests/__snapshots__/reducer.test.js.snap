// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`languageProviderReducer should handle the SET_LOCALE action correctly 1`] = `
Object {
  "locale": undefined,
}
`;
